README.md

```diff
@@ -164,7 +164,9 @@ An `or` combiner can use the keyword `or` as well as `,`.
 `last 1 version or > 1%` is equal to `last 1 version, > 1%`.
 
 `and` query combinations are also supported to perform an
-intersection of the previous query: `last 1 version and > 1%`.
+intersection of all the previous queries: `last 1 version or chrome > 75 and > 1%`,
+which in human words matches:
+`last 1 version of any browser with more than 1% of the market share and every version of chrome higher than 76 with more than 1% market share`.
```

Comment:
Multiline queries do not look beautiful for me

![20200701203806](https://user-images.githubusercontent.com/19343/86313791-d63fee00-bbda-11ea-99f5-a5a5cc7bab59.png)

Can we use a quote and simplify the example, because it is hard to understand so long sentence.

Reply:
Changed it to pseudocode. I think this would make more sense for developers and dev related people who are the audience.

```
which in pseudo code matches:
`match = (is browser last version OR is chrome since v76) &&  has more than 1% marketshare`
```